component/header.jsx

```diff
@@ -1,18 +1,22 @@
 import styled from 'styled-components';
+import { FaSwatchbook } from 'react-icons/fa';
 
 export default function Header() {
   return (
     <StyledHeader>
-      <img src={'/icons8-study.png'} />
-      <h1>My Book Tracking App</h1>
+      <h1>
+        <FaSwatchbook style={{ color: '#4a82c2', fontSize: '60px' }} /> My Book
+        App
+      </h1>
```

Comment:
I'm not quite sure about this. Maybe I would prefer putting the SVG outside the `h1` because it's not a text/heading but a logo. If you do so, you could still easily position them with styles on the `StyledHeader`.